enclave_wrangler/objects_api.py

```diff
@@ -353,8 +356,17 @@ def fetch_cset_and_member_objects(
     for cset in cset_versions:
         version: int = cset['properties']['codesetId']
         # Hierarchical
-        cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
-        cset['member_items']: List[Dict] = get_concept_set_version_members(version, return_detail='full')
+        # TODO: exc handling: Will need to handle differently or do more: https://github.com/jhu-bids/TermHub/issues/451
+        try:
+            cset['expression_items']: List[Dict] = get_concept_set_version_expression_items(version, return_detail='full')
+        except EnclavePaginationLimitErr as err:
+            cset['expression_items']: List[Dict] = err.args[1]['results_prior_to_error']
```

Comment:
Mind you, this is just temporary as I don't have time for a better fix today. I'd like to get your feedback on my possible solutions here:
- #451